docs/guide/getstarted/configuration.md

```diff
@@ -97,6 +97,12 @@ Saves a screenshot to a given path if the Selenium driver crashes
 Type: `String`|`null`<br>
 Default: *null*
 
+### screenshotOnReject
+Attaches a screenshot of a current page to the error if the Selenium driver crashes
+
+Type: `Boolean`<br>
+Default: *false*
+
```

Comment:
Why can't we just attach all screenshots to the error? Do you see any performance issues?